frontend/src/types/Uid.ts

```diff
@@ -0,0 +1,3 @@
+import { Brand } from './Brand';
+
+export type Uid = Brand<'Uid', string>;
```

Comment:
Since you are also using this in the backend, maybe you should create a shared folder for all shared code